### PR TITLE
feat: add iam submodule for tag keys and values

### PR DESCRIPTION
**Pull request by imrannayer**

### CODE REVIEW

File: examples/bigquery_dataset/main.tf [hunk 1]
```diff
@@ -47,3 +47,8 @@ resource "google_bigquery_dataset" "bigquery_dataset_one" {
 resource "random_id" "test" {
   byte_length = 4
 }
+
+output "bigquery_dataset_one" {
```
Comment (bharathkkb) [inline examples/bigquery_dataset/main.tf:51]: I didn't grok the reason for adding these outputs

Comment (imrannayer) [reply examples/bigquery_dataset/main.tf:51]: Lint tests are failing of there is no output thats why I had to add outputs to the examples which were missing outputs. It is strange that these tests are failing in cloudbuild. When I run tests locally they dont show as an issue.

**Review by bharathkkb — APPROVED** [review]

**Pull request merged**